### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by MarcelGerber**
cc @ingorichter @pthiess

### CODE REVIEW

File: updates/stable/de.json [hunk 1]
```diff
@@ -7,20 +7,20 @@
         "downloadURL": "http://brackets.io",
         "newFeatures": [
             {
-                "name": "Command Line Integration",
-                "description": "Launch Brackets and open files/folders from the command line on Mac or Windows. Brackets can also integrate with the Windows Explorer context menu. On Windows these are installer options; on Mac, use File > Install Command Line Shortcut to enable."
+                "name": "Kommandozeilenintegration",
+                "description": "Nutzen Sie die Kommandozeile, um Brackets unter Windows und auf dem Mac zu starten und um Dateien oder Ordner zu öffnen. Brackets lässt sich auch in das Kontextmenü vom Windows Explorer integrieren. Unter Windows können diese Optionen bei der Installation einzeln ausgewählt werden; auf dem Mac können sie via Datei > Kommandozeilenverknüpfung einrichten aktiviert werden."
             },
             {
                 "name": "Code Folding",
```
Comment (MarcelGerber) [inline updates/stable/de.json]: I've gone with the English one as all the alternatives (`Code-Faltung`, `Code-Falten`, `Code-Folding`) sound rather awkward imho.

**Comment by nethip** [comment]
@ingorichter @pthiess Could you guys take some time in reviewing this PR. Thanks!

**Comment by pthiess** [comment]
LGTM, I'm not sure if this should go into master or the release branch - that's the only reason I'm not merging yet. @MarcelGerber @nethip please feel free to go ahead.

File: updates/stable/de.json (hunk 1 above)
Comment (pthiess) [inline updates/stable/de.json]: +1

**Comment by nethip** [comment]
@pthiess Thanks for reviewing this :smile:

**Pull request merged**